src/tests/suite/SConscript

```diff
@@ -107,7 +107,6 @@ def scons():
                     daos_aggregate_ec.c daos_degrade_ec.c
                     daos_extend_simple.c daos_obj_ec.c""")
 
-    newenv.AppendUnique(LIBPATH=["$BUILD_DIR/src/client/dfs"])
```

Comment:
i see on the master PR that this was moved to earlier, but here in the 2.2 one it's just removed. can you please explain why?

Reply:
I thought it could inherit from parent Sconscript, see it was appended in src/tests/Sconscript modification.

Reply:
ah ok i see. i guess i was confused why this change was made then only for 2.2 and not master. but ok we can unblock this. probably better to keep master in sync to make future cherrypicking to the branches easier.